src/main/scala/mesosphere/marathon/core/task/tracker/InstanceTracker.scala

```diff
@@ -28,6 +28,20 @@ trait InstanceTracker {
   def specInstancesSync(pathId: PathId): Seq[Instance]
   def specInstances(pathId: PathId)(implicit ec: ExecutionContext): Future[Seq[Instance]]
 
+  /**
+    * Get all instances for a run spec with given id.
+    *
+    * @param pathId The id of the run spec.
+    * @return All instances for the run spec.
+    */
+  def get(pathId: PathId)(implicit ec: ExecutionContext): Future[Seq[Instance]] = specInstances(pathId)
```

Comment:
Why did you decide to add a `get` that returns a list based on the given Path?

Given that the entity the InstanceTracker handles is an instance, the more natural (?) approach _to me_ seems to be
```
def get(instanceId: Instance.Id): Future[Option[Instance]]
def list(pathId: PathId): Future[Seq[Instance]]
```

WDYT?

Reply:
Good point. I don't event think I use the `get` so far. I'll make the change.

Reply:
Done ✅